app-text/discount/discount-2.2.1.recipe

```diff
@@ -21,27 +21,27 @@ PROVIDES="
 	cmd:theme = $portVersion compat >= 2
 	"
 REQUIRES="
-	haiku
+	haiku$secondaryArchSuffix
```

Comment:
**`$secondaryArchSuffix`** is only useful for recipes that define a SECONDARY_ARCHITECTURES.
Since both conditions are met:
* this recipe provides only commands (**`cmd:theme`**) but no library (**`lib:` or **`devel:`**);
* this recipe builds for all primary architectures;

we don't need to make it build for secondary architectures.
So having **`$secondaryArchSuffix`** is not needed (nor recommended) because it will always be the empty.